src/account.rs

```diff
@@ -92,6 +97,27 @@ pub(crate) struct Balance {
     unverified: Unverified,
 }
 
+#[derive(Debug, Args)]
+pub(crate) struct Transfer {
+    /// The bech32 address (in Bech32Address) of the account to transfer assets to.
+    #[clap(long)]
+    to: Bech32Address,
```

Comment:
> to possibly support using account indexes instead

Ah yes, maybe we can add a `--to-account <ix>` flag at some point?

On that topic, perhaps rather than only supporting `Bech32Address` here, we should also support hex addresses? Can we create a wrapper `enum` type around the hex `Address` and `Bech32Address` types that `impl`s `FromStr` and use that here instead? Would be great if users didn't have to think about what type of address they had to pass here.